volunteer/views.py

```diff
@@ -4,4 +4,5 @@
 
 @group_required("volunteer")
 def volunteer(request):
+    #  New comment here
```

Comment:
Why?

Reply:
Oops, I think thats a vestige that accidentally got pushed from my teaching Ernesto about Python and Django.